packages/lodestar/src/sync/utils/sync.ts

```diff
@@ -65,6 +67,22 @@ export function isValidFinalizedCheckPoint(peers: IReputation[], finalizedCheckp
   return peerCount >= (validPeers.length / 2);
 }
 
+export function isValidPeerForInitSync(config: IBeaconConfig, myState: BeaconState, peerStatus: Status): boolean {
+  if (!peerStatus) {
+    return false;
+  }
+  if (!config.types.Version.equals(myState.fork.currentVersion, peerStatus.headForkVersion)) {
```

Comment:
We should remove this check as our peer might be on different fork version after block x but it's still valid for block 0..x
I think this is specified in latest spec release

Reply:
yeah, just saw we have to check something called `fork_digest`. Should mark a TODO here instead.